solutions/pandas_tests/pytest.ini

```diff
@@ -0,0 +1,11 @@
+# pytest.ini]
```

Comment:
What's the importance of the defined markers in this file? Is this file from pandas repo?

Reply:
These are markers defined in the pandas test suite and are not registered (suprisingly) in the initialization file within the pandas/tests/. Introducing this file removed many of the "marker not found" errors when initializing the tests.

Reply:
simply put
```suggestion
# pytest.ini
```

Reply:
I guess the problem is:
- Do we know what's the set of markers that pandas tests use?
- If so, is there a reference?
- Or do we get the following list by trial-and-error?

Reply:
I had to use trial and error to compose this list of markers in pandas/tests/. There is no one place where all these markers are listed..